tomcat/src/main/java/org/apache/catalina/connector/Connector.java

```diff
@@ -1,30 +1,34 @@
 package org.apache.catalina.connector;
 
-import org.apache.coyote.http11.Http11Processor;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
 import java.io.IOException;
 import java.io.UncheckedIOException;
 import java.net.ServerSocket;
 import java.net.Socket;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import org.apache.coyote.http11.Http11Processor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class Connector implements Runnable {
 
     private static final Logger log = LoggerFactory.getLogger(Connector.class);
 
     private static final int DEFAULT_PORT = 8080;
     private static final int DEFAULT_ACCEPT_COUNT = 100;
+    private static final int DEFAULT_MAX_THREADS = 250;
 
     private final ServerSocket serverSocket;
+    private final ExecutorService executorService;
     private boolean stopped;
 
     public Connector() {
-        this(DEFAULT_PORT, DEFAULT_ACCEPT_COUNT);
+        this(DEFAULT_PORT, DEFAULT_ACCEPT_COUNT, DEFAULT_MAX_THREADS);
     }
 
-    public Connector(final int port, final int acceptCount) {
+    public Connector(int port, int acceptCount, int maxThreads) {
```

Comment:
```suggestion
    public Connector(final Container container, final int port, final int acceptCount, final int maxThreads) {
}
```

lms에 Container에 대한 요구사항이 자세하게 나와있지만 릴리가 생각하는 Container는 무엇일지 궁금해요
꼭 구현하지 않아도 고민해보면 좋을 포인트인 것 같아 리뷰 남깁니다...!

Reply:
톰캣 같은 서블릿 컨테이너에서 Container는 웹 컴포넌트들을 관리하고 HTTP 요청을 처리하는데요,
현재는 HTTP 요청을 처리하는 책임을 Http11Processor 가 가지고 있어 Container가 필요하지 않다고 생각했습니다. 
대신 executorService 가 스레드 풀 관리를 해주고 있고요!